bigbluebutton-client/src/org/bigbluebutton/modules/chat/services/ChatMessageService.as

```diff
@@ -125,11 +129,11 @@ package org.bigbluebutton.modules.chat.services
           moderatorOnlyMsg.toUserID = SPACE;
           moderatorOnlyMsg.toUsername = SPACE;
           moderatorOnlyMsg.message = MeetingModel.getInstance().modOnlyMessage;
-          
+          
           var moderatorOnlyMsgEvent:PublicChatMessageEvent = new PublicChatMessageEvent(PublicChatMessageEvent.PUBLIC_CHAT_MESSAGE_EVENT);
           moderatorOnlyMsgEvent.message = moderatorOnlyMsg;
           moderatorOnlyMsgEvent.history = false;
-          dispatcher.dispatchEvent(moderatorOnlyMsgEvent);
+          dispatcher.dispatchEvent(moderatorOnlyMsgEvent);

```

Comment:
Usually we clear this kind of modifications before the PR, and keep only the valuable modifications.